Protocol.md

```diff
@@ -639,7 +648,8 @@ key pair (public key as part of the WebSocket path or sent in
   "subprotocols": [
     "v1.saltyrtc.org",
     "some.other.protocol"
-  ]
+  ],
+  "ping_interval": 30  // optional
```

Comment:
I'd make that field non-optional (easier to parse), but the client can set the value to `0` to indicate that no ping messages are requested. This would mirror the format used for chunk size in the WebRTC task.